pom.xml

```diff
@@ -6,7 +6,15 @@
   <groupId>com.azure</groupId>
   <artifactId>azure-sdk-all</artifactId>
   <packaging>pom</packaging>
-  <version>1.0.0</version> <!-- {x-version-update;com.azure:azure-sdk-all;current} -->
+  <version>1.0.0</version>
+    <dependencies>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-core</artifactId>
+            <version>1.4.0-beta.2</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies> <!-- {x-version-update;com.azure:azure-sdk-all;current} -->
```

Comment:
This will add Azure Core as a dependency to all modules depending pom.xml, this dependency should be removed.

Reply:
Wrong place to add. Will remvoe.